Users/Tutorial.rst

```diff
@@ -28,7 +28,7 @@ use with this sample code and may need minor adjustments.
 Let's Start!
 ------------
 
-There are two options how to let coala know what kind of analysis it
+There are two options used to let coala know what kind of analysis it
```

Comment:
You're right, the full sentence sounds awkward. How about: 
"coala has two options for code analysis" ?